pkg/server/registry/apigroups/acorn/projects/validator.go

```diff
@@ -87,14 +87,28 @@ func (v *Validator) ensureNoObjectsExistInRegions(ctx context.Context, namespace
 				field.NewPath("spec", "supportedRegions"),
 				regions,
 				fmt.Sprintf(
-					"cannot remove regions %v that have %s: %v",
+					"cannot remove regions %v while in use by the following %ss: %v",
 					regions,
-					strings.TrimSuffix(strings.ToLower(obj.GetObjectKind().GroupVersionKind().Kind), "list"),
```

Comment:
If the `obj` passed here has no GVK set on it's typemeta -- which is what we're doing -- then this evaluates to an empty string.